src/Adapters.Tests/DiagnosticSource.Tests/DiagnosticSourceListener.Tests.csproj

```diff
@@ -18,19 +18,17 @@
     </PackageReference>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\Adapters\DiagnosticSource\DiagnosticSourceListener.csproj" />
+  </ItemGroup>
+
   <ItemGroup>
     <!-- Identifies the project as test project -->
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'Test.props'))\Test.props" />
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\Adapters\DiagnosticSource\DiagnosticSource.csproj">
-      <IncludeAssets>All</IncludeAssets>
```

Comment:
Do we need this in new reference as well?

Reply:
we do not really need `<IncludeAssets>All</IncludeAssets>` as this is default. I'll remove it everywhere